DEVELOPER_GUIDE.md

```diff
@@ -51,24 +55,27 @@ Example output: `./build/anomaly-detection-dashboards-1.0.0.0.zip`
 
 In the base OpenSearch Dashboards directory, run
 
-- `yarn start --no-base-path`
+`yarn start --no-base-path`
 
-  Starts OpenSearch Dashboards and includes this plugin. OpenSearch Dashboards will be available on `localhost:5601`.
+Starts OpenSearch Dashboards and includes this plugin. OpenSearch Dashboards will be available on `localhost:5601`.
 
 ### Test
 
-- `yarn test:jest`
+#### Unit Tests
 
-  Runs the plugin unit tests.
+`yarn test:jest`
 
-- `yarn test:e2e`
+Runs the plugin unit tests.
 
-  Start OpenSearch Dashboards, wait for it to be available on `localhost:5601`, and runs end-to-end tests.
+#### Integration Tests
 
-- `yarn cy:run`
+Integration tests for this plugin are written using the Cypress test framework. The tests are maintained in the central [opensearch-dashboards-functional-test](https://github.com/opensearch-project/opensearch-dashboards-functional-test) repository. Steps to run the tests:
 
-  Runs end-to-end tests on a currently running OpenSearch Dashboards server. Defaults to run the tests on `localhost:5601`, although you can change this to run on any
-  OpenSearch Dashboards server with the command `yarn cy:run --config baseUrl=<your-custom-URL>`
+1. Checkout [opensearch-dashboards-functional-test](https://github.com/opensearch-project/opensearch-dashboards-functional-test)
+2. Spin up a cluster with the Anomaly Detection Dashboards Plugin installed
+3. In the root directory, run a command using [Cypress CLI](https://docs.cypress.io/guides/guides/command-line)
```

Comment:
In the root directory of opensearch-dashboards-functional-test? For development then we should build AD-dashboards with latest code and then add tests to functional-test repo and run the tests right?

Reply:
Right, that's the strategy. Good catch on the wording. I'll update to make it clear

Reply:
Updated in https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/174/commits/875ae40a4c972d57f68103af42a417b4b9e5e183